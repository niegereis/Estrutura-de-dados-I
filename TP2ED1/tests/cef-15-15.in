15 15
f
***************
*   *   *     *
*** * * * *** *
*M* * *     * *
* * *** ***** *
* *   * *     *
* *** * * *****
*     * * *   *
* ***** * * * *
* *     *   * *
* *** ******* *
*   *     *   *
*** ******* ***
*           *  
***************