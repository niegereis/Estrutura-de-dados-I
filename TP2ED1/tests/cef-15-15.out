94
***************
*ooo*ooo*ooooo*
***o*o*o*o***o*
*M*o*o*ooooo*o*
*o*o***o*****o*
*o*ooo*o*ooooo*
*o***o*o*o*****
*ooooo*o*o*ooo*
*o*****o*o*o*o*
*o*ooooo*ooo*o*
*o***o*******o*
*ooo*ooooo*ooo*
***o*******o***
*ooooooooooo*  
***************
